deploy/yamls/zfs-driver.yaml

```diff
@@ -805,6 +805,8 @@ spec:
               value: "zfs-operator"
             - name: OPENEBS_IO_ENABLE_ANALYTICS
               value: "true"
+            - name: NODE_AFFINITY_KEY
+              value: kubernetes.io/hostname
```

Comment:
Is this the label key that we are using? It is added to each node by kubernetes right? So how does the labelling a new node work when another node goes down?

Reply:
it is the default key used by the driver. User has to label and provide that key here. See the docs added.

Reply:
If the default key is used, PV migration cannot be done right?

Reply:
Yep